aexpect/client.py

```diff
@@ -1093,7 +1093,7 @@ def cmd_output_safe(self, cmd, timeout=60):
                 o += self.read_up_to_prompt(0.5)
                 success = True
                 break
-            except ExpectError, e:
+            except ExpectError as e:
```

Comment:
OK, I have an ack from @xutian, they use autotest for RHEL5 testing. So let me re-evaluate this PR and merge it...